server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_2__notification_policy_metadata.yaml

```diff
@@ -0,0 +1,991 @@
+kind: resource
```

Comment:
this script will be run for every organization and there is not unique key exists and may cause fatal error

Reply:
Apologize for my misunderstanding of the usage of the migration framework. I moved it into `R4242NotificationPolicyMetaMigrate` and `notification-policy-metadata.yaml`. This will only be executed once.